### PR TITLE
dash/getoptions-json-serialization

### PR DESCRIPTION
**Pull request by sebastianbochan**
- [x] - [Save state after resizing ](https://app.asana.com/0/0/1204755770575894) - https://github.com/highcharts/highcharts/pull/19241
- [x] - [Save state after drag cell](https://app.asana.com/0/0/1204755770575895) - https://github.com/highcharts/highcharts/pull/19239
- [x] - [Save state after adding new component ](https://app.asana.com/0/0/1204755770575898) https://github.com/highcharts/highcharts/pull/19318
- [x] - [Save state after update from sidebar](https://app.asana.com/0/0/1204755770575896) - https://github.com/highcharts/highcharts/pull/19252
- [x] - [Make sure the removing cells/component is reflected in the JSON ](https://app.asana.com/0/0/1204755770575900) https://github.com/highcharts/highcharts/pull/19254
- [x] - [Change toJSON method for each element like Board, Layout, Row, Cell, Component](https://app.asana.com/0/0/1204755770575893) https://github.com/highcharts/highcharts/pull/19307
  - [x] - Board
  - [x] - Layout
  - [x] - Row
  - [x] - Cell
  - [x] - HTML Component
  - [x] - DataGrid Component
  - [x] - Highcharts Component
  - [x] - KPI Component
  - [x] - doclets
- [x] - [Create one Cypress test with all interactions](https://app.asana.com/0/0/1204755770575904) https://github.com/highcharts/highcharts/pull/19318
- [x] - [Demo where the JSON is shown in the textarea](https://app.asana.com/0/0/1204755770575903) https://github.com/highcharts/highcharts/pull/19247

### CODE REVIEW

File: ts/Dashboards/Board.ts [hunk 1]
```diff
@@ -623,6 +623,41 @@ class Board implements Serializable<Board, Board.JSON> {
         };
     }
 
+    /**
+     * Convert the current state of board's options into JSON. The function does
+     * not support converting functions or events into JSON object.
+     *
+     * @returns
```
Comment (karolkolodziej) [inline ts/Dashboards/Board.ts:630]: Please add link to the demo.

**Review by karolkolodziej — APPROVED** [review]
Great work! 

Please consider changing the demo to match the one from the chart- [API](https://api.highcharts.com/class-reference/Highcharts.Chart#getOptions)

**Review by pawellysy — APPROVED** [review]
Great job!

**Review by bre1470 — APPROVED** [review]
Nice! Looks good. 👍

**Review by pawelfus — APPROVED** [review]
Amazing, thanks!

**Pull request merged**